addons/a11y/src/a11yHighlight.ts

```diff
@@ -0,0 +1,32 @@
+import { document } from 'global';
+import addons from '@storybook/addons';
+import { EVENTS } from './constants';
+
+if (module && module.hot && module.hot.decline) {
+  module.hot.decline();
+}
+
+interface HighlightInfo {
+  /** html selector of the element */
+  elements: string[];
+  color: string;
+}
+
+const channel = addons.getChannel();
+
+const highlight = (infos: HighlightInfo) => {
+  const id = 'a11yHighlight';
+  const sheetToBeRemoved = document.getElementById(id);
+  if (sheetToBeRemoved) {
+    sheetToBeRemoved.parentNode.removeChild(sheetToBeRemoved);
+  }
+
+  const sheet = document.createElement('style');
+  sheet.setAttribute('id', id);
+  sheet.innerHTML = infos.elements
+    .map((target) => `${target}{ outline: 1px dotted ${infos.color}!important; }`)
```

Comment:
Ow nice, so the color in dependent on the priority?

Real meta, but how do we make sure those outlines are actually visible / accessible?

We should add a box-shadow perhaps of 1px background behind the dotted  outline?

Reply:
That's actually a good question. I'm not a big fan of the outline, I can't see the difference between red / green. Maybe we can have this is the a11y parameters ? (different PR), not sure how the theme impacts this

Reply:
Assume no theming for now, just static colors.

Reply:
What do you think of this ?
![image](https://user-images.githubusercontent.com/599163/79693811-290d3900-826d-11ea-9aca-6cb1579dd896.png)

Reply:
Made a tweak to this, WDYT?

Reply:
Good for me ;)